docs/source/release-notes.rst

```diff
@@ -3,6 +3,22 @@ FiftyOne Release Notes
 
 .. default-role:: code
 
+FiftyOne Teams 1.6.0
+--------------------
+*Released March 29, 2024*
+
+What's New
```

Comment:
I'm going to add links throughout this section to the CAS doc. I will also need to add more content that matches to do so.